.gitlab-ci.yml

```diff
@@ -888,6 +888,14 @@ library:ci-trakt:
   - plugin:ci-elpi_hb
   stage: build-2
 
+library:ci-wasm:
+  extends: .ci-template
+  needs:
+  - library: ci-compcert
+  - library: ci-ext-lib
+  - library: ci-mathcomp
```

Comment:
these dependencies should also appear in Makefile.ci

Reply:
Thanks!
In fact, there's also a dependency on parseque, but I couldn't find the relevant ci here (there's a file called ci-parsec, but it's not used anywhere in the main CI file).